background/keepass.js

```diff
@@ -194,18 +199,22 @@ Keepass.reCheckAssociated = function () {
   });
 };
 
-Keepass.getLogins = function (url, callback, errcallback) {
+Keepass.getLogins = function(url) {
   if (!this.ready()) {
-    Keepass.associate(function () {
-      Keepass.getLogins(url, callback, errcallback);
+    return Keepass.associate().then(() => {
+      return Keepass.getLogins(url);
```

Comment:
This can be condensed to an anonymous function without braces and return
(also reported by the linter)